cirq-google/cirq_google/devices/grid_device_test.py

```diff
@@ -644,6 +679,7 @@ def test_to_proto():
             cirq.ops.FSimGate, tags_to_accept=[cirq_google.FSimViaModelTag()]
         ): base_duration
         * 10,
+        cirq.GateFamily(cirq.CZPowGate): base_duration * 11,
```

Comment:
just to double check CZ will take `11ns`, right?

Reply:
I was confused about this test until I stared at it for a while, so just pointing out a couple of things:

- These gate durations are arbitrary; take a look at lines 118-151, and you'll see that they are just arbitrary durations, and then the test compares them to `gate_durations = [(n, i * 1000) for i, n in enumerate(gate_names)]` (line 83).
- In this test (not in real life), cz is given a duration of 3 ns (line 127).